lib/utils/frc_teams.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:neatteam_scouting_2023/models/team.dart';
+import 'package:neatteam_scouting_2023/utils/utils.dart';
+
+class FrcTeams {
+  static Future<List<Team>> getAllTeams() async {
```

Comment:
API will look like `FrcTeams.getAllTeams()`, "team" is duplicated.

```suggestion
  static Future<List<Team>> getAll() async {
```